packages/types/mongoose-store/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as Promise from 'bluebird';
+import * as mongoose from 'mongoose';
+
+export default class MongooseStore<T>{
+  constructor(protected conn: mongoose.Connection) {
```

Comment:
I think that passing the connection from outside gives the app better control of its own initialization and makes it easier for sharing it between different `MongooseStore` instances.